README.md

```diff
@@ -1,2 +1,8 @@
-# messaging-java
+# CMO Messaging Library
+
 The messaging library used by Java clients to interface with CMO messaging backbone.
+
+Properties for build:
+
+- `nats.connection_url` : connection URL to NATS server
+- `nats.flush_duration` : Flush duration in seconds. Default is 5 seconds.
```

Comment:
Is this the same as a connection/socket timeout? I'm wondering if its possible to set some timeout when attempting to publish or subscribe that could cause a blocking of the client until the timeout is exceeded.

Reply:
I believe so from what i understand. 

>Many of the client libraries use the PING/PONG interaction built into the NATS protocol to ensure that flush pushed all of the buffered messages to the server. When an application calls flush, most libraries will put a PING on the outgoing queue of messages, and wait for the server to respond with a PONG before saying that the flush was successful.